Makefile

```diff
@@ -46,7 +46,7 @@ stop-nginx:
 
 .PHONY: start-nginx
 start-nginx:
-	docker run --rm --name "${DOCKER_IMAGE_NAME}" -d -p 8000:8000 ${DOCKER_ORG_NAME}/${DOCKER_IMAGE_NAME}
+	docker run --rm --name "${DOCKER_IMAGE_NAME}" -d -p 8000:80 ${DOCKER_ORG_NAME}/${DOCKER_IMAGE_NAME}
```

Comment:
Why the port change?

Reply:
It was in the commit message :].

Because the nginx config of the built Docker image, is based on nginx' default config, where it listens on port 80.

Reply:
Oh, sorry I missed that. Thanks!